Lib/inspect.py

```diff
@@ -291,12 +291,17 @@ def isbuiltin(object):
         __self__        instance to which a method is bound, or None"""
     return isinstance(object, types.BuiltinFunctionType)
 
+def ismethodwrapper(object):
+    """return True if the object is a method-wrapper."""
```

Comment:
Since method wrappers are a rather esoteric concept, this doc-string should include more details about them, similar to the doc-strings for `ismethoddescriptor()` and `isdatadescriptor()`.

Reply:
I tried to add more details and give this address https://docs.python.org/3/library/types.html#types.MethodWrapperType but I am not sure I am applying the correct method, do you have any suggestions?